lib/sles4sap/ipaddr2.pm

```diff
@@ -358,6 +358,50 @@ sub ipaddr2_deployment_sanity {
         $count = grep(/running/, @$res);
         die "VM $vm is not fully running" unless $count eq 2;    # 2 is two occurrence of the word 'running' for one VM
     }
+}
```

Comment:
apparently there was a missing close `}` in previous code that was there unnoticed :-(